releasewarrior/__main__.py

```diff
@@ -75,6 +75,8 @@ def parse_args():
                                help='taskcluster graphid used for release in question')
     parser_update.add_argument('--graphid-2', '--taskcluster-graphid-2', dest='graphid_2',
                                help='taskcluster graphid used for graph 2 of rc release in question')
+    parser_update.add_argument('--checkbox', choices=ALL_CHECKBOXES, dest='checkboxes',
```

Comment:
will this be confusing when you `--help` to see all choices and it returns every choice including aliases in one list?

e.g. 'submitted_shipit' and 'shipit' both in choices even though they are the same?

I'm wondering if we should get rid of aliases?